### DISCUSSION TITLE
Pathway to Riverscapes Compliant status for Fluvial Corridor Toolbox

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-tools#811] [Q&A]
@MattReimer, now that you have your new PIP module done, can you lay out next steps (revisiting from November) for @sdunesme to use that to write Riverscapes Projects from FCT please? Feel free to move to more appropriate location if you don't like here.

**Comment by philipbaileynar** [comment]
@joewheaton there are some big question marks about whether FCT can ever be riverscapes compliant while it uses VRTs instead of actual raster files.

That said, the technical task of writing a `project.rs.xml` file is straightforward with the new pip module. The repo [readme](https://github.com/Riverscapes/RiverscapesXML/tree/master/python/packages/rsxml) file includes documentation. There are also several [examples](https://github.com/Riverscapes/RiverscapesXML/tree/master/python/packages/rsxml/examples) of how to use the various libraries.

The hardest part for any tool developer wanting to incorporate their model into the riverscapes world is designing how to structure their project files. There are dozens of decisions that don't have right/wrong answers that are bespoke to each model. Documentation for this would be extremely difficult and take a very long time to perfect.
